basics/trey-hunner-zip-and-enumerate.md

```diff
@@ -153,28 +131,16 @@ Roll number of Michel is 28
 
 `enumerate()` is an amazing Built-in function offered by python. When used, gives us the index and the item combined. long story short! `enumerate` comes in handy while working with loops as it gives us access to both the item and it's index at the same time.
 
-### Syntax
-
-`enumerate(iterable, start=0)`
-
-### Parameters
-
-* Iterable: any iteration supporting object
-* Start: Starting index value, default is 0
-
-### Example 1: basic enumerate example
-
 ```python
 >>> even_nums = [2, 4, 6, 8, 10, 12]
-print(list(enumerate(even_nums)))
->>> 
+>>> print(list(enumerate(even_nums)))
 [(0, 2), (1, 4), (2, 6), (3, 8), (4, 10), (5, 12)]
 >>>
 ```
 
-Now before we use it in a loop let's take a look how we would do the same thing in a traditional way.
+Before we use it in a loop let's take a look how we would do the same thing in a traditional way.
 
-### Example 2: looping without `enumerate()`
+### looping without `enumerate()`
```

Comment:
This is another code-only section. We should explain:
- `range(0, len(even_nums))` gives 0,1,2,3,4,5, with the list length 6 excluded
- Because these are the valid indexes of a list of length 6, it prints each element of `even_nums`
- This is complicated to think about and easy to get wrong, so it's better to use `enumerate()`